system/athena/athenad.py

```diff
@@ -560,6 +560,22 @@ def getNetworks():
   return HARDWARE.get_networks()
 
 
+@dispatcher.add_method
+def getEsimProfiles():
+  return [asdict(p) for p in HARDWARE.get_sim_lpa().list_profiles()]
+
+
+@dispatcher.add_method
+def downloadEsimProfile(lpa_activation_code: str, profile_name: str):
+  HARDWARE.get_sim_lpa().download_profile(lpa_activation_code, profile_name)
+
+
+@dispatcher.add_method
+def setEsimProfile(iccid: str):
```

Comment:
i want to do the input validation of any "write" operations inside the athena method here